### ISSUE TITLE
BeagleBone green

### ISSUE THREAD

**Issue by Fabricio-Valerio** [beagleboard/bone101#25]
Hello, I've started to use a BeagleBone Green Board, I'm following the steps from https://beagleboard.org/getting-started, my PC is connected to the board via USB but a network connection hasn't showed up, and I'm unnable to access the BeagleBone. What can I do?

Thanks in advance.

**Comment by RobertCNelson** [comment]
Are you using Windows, Mac or Linux (what version)?

**Comment by Fabricio-Valerio** [comment]
> Are you using Windows, Mac or Linux (what version)?

I'm Using Windows 8.1

**Comment by RobertCNelson** [comment]
Just grab the latest iot image:

https://beagleboard.org/latest-images

![latest](https://user-images.githubusercontent.com/91469/98851205-8e986100-241b-11eb-9eca-7ae48e243306.png)

Use etcher to write the microSD: https://www.balena.io/etcher/

Regards,

**Comment by Fabricio-Valerio** [comment]
> Just grab the latest iot image:
> 
> https://beagleboard.org/latest-images
> 
> ![latest](https://user-images.githubusercontent.com/91469/98851205-8e986100-241b-11eb-9eca-7ae48e243306.png)
> 
> Use etcher to write the microSD: https://www.balena.io/etcher/
> 
> Regards,

I've already did that. There is a way to check if I did the proccess correctly?

![image](https://user-images.githubusercontent.com/33754194/98851843-35d1c400-242d-11eb-9804-db29b62c0d65.png)

The image shows what happens when I connect the board via USB.

**Comment by RobertCNelson** [comment]
The Date's are 2016 in that directory.. HOLD the boot button before you plug in power.. Your boot off the eMMC..

Regards,

**Comment by Fabricio-Valerio** [comment]
> The Date's are 2016 in that directory.. HOLD the boot button before you plug in power.. Your boot off the eMMC..
> 
> Regards,

The boot button do you mean S2 - USER button?

**Comment by RobertCNelson** [comment]
Correct user.

**Comment by Fabricio-Valerio** [comment]
Now the board is like that: 


![image](https://user-images.githubusercontent.com/33754194/98854762-5d2a9000-2431-11eb-9015-14ba53638166.png)
The First and the third led are blinking.

**Comment by RobertCNelson** [comment]
first is heartbeat, 3rd should be microsd... check your device manager..

**Comment by Fabricio-Valerio** [comment]
My device manager:
![image](https://user-images.githubusercontent.com/33754194/98855382-489ac780-2432-11eb-9f4c-f3a0ad3e3577.png)
Is it correct?

**Comment by RobertCNelson** [comment]
yeah, looks correct, if you have windows 8.1's windows update working it should pull down microsoft's driver..

**Comment by Fabricio-Valerio** [comment]
> yeah, looks correct, if you have windows 8.1's windows update working it should pull down microsoft's driver..

My windows update was deactivated, now it's running.

**Comment by Fabricio-Valerio** [comment]
This image shows the content when the SD card was in the board.
![image](https://user-images.githubusercontent.com/33754194/98861959-7dac1780-243c-11eb-93ca-f9c716873fa1.png)
I've waited 45 minutes and then disconnected the board from my PC, then when I turned it on again, it was with these files again.
![image](https://user-images.githubusercontent.com/33754194/98862236-f0b58e00-243c-11eb-8a2a-42261580e2ae.png)
And with the SD Card in the board the network adapter showed up and I was able to ping
![image](https://user-images.githubusercontent.com/33754194/98862352-2490b380-243d-11eb-82a2-0136223e9650.png)
![image](https://user-images.githubusercontent.com/33754194/98862409-3ffbbe80-243d-11eb-93eb-0b4f0286110d.png)

**Comment by RobertCNelson** [comment]
Why did you wait 45 Minutes?

Your new image was already running...

Regards

**Comment by Fabricio-Valerio** [comment]
> Why did you wait 45 Minutes?
> 
> Your new image was already running...
> 
> Regards

But if I remove the SD Card the new image would be still running?

**Comment by RobertCNelson** [comment]
No, if you remove the microSD, it'll boot the old image on the eMMC.. As you removed the microSD..

Like I asked, why did you wait 45 minutes, are you expecting it to do something for you?

Regards,

**Comment by Fabricio-Valerio** [comment]
I expected to replace the older image by the newest one. Can I do that?
Because my application is going to run in many boards, and I can't (or wasn't expecting) use SD card in each one.
Regards.

**Comment by RobertCNelson** [comment]
That's called a "flasher" image:

![flasher](https://user-images.githubusercontent.com/91469/98863705-5f3f1f80-242e-11eb-97bf-5d9117d710c2.png)

**Comment by RobertCNelson** [comment]
You can convert any non-flasher image to a flasher via:

https://elinux.org/Beagleboard:BeagleBoneBlack_Debian#Flashing_eMMC

This allows you to customize the microSD image, before you start flashing a bunch of boards..

Regards,

**Comment by Fabricio-Valerio** [comment]
> You can convert any non-flasher image to a flasher via:
> 
> https://elinux.org/Beagleboard:BeagleBoneBlack_Debian#Flashing_eMMC
> 
> This allows you to customize the microSD image, before you start flashing a bunch of boards..
> 
> Regards,
Oh, so this "flasher" image allows me to replace the old one, after removing the SD card?

**Comment by RobertCNelson** [comment]
The flasher will copy the live image from the microSD to the eMMC..

In most cases it'll take around 10 minutes and shut down the board when complete.

**Comment by Fabricio-Valerio** [comment]
Ok, I'll try it, thank you very much Mr.Robert.

**Comment by Fabricio-Valerio** [comment]
I've created a File named example.cpp in the directory BeagleBone/Green, and after that the cloud IDE keeps loading:
![image](https://user-images.githubusercontent.com/33754194/98954740-7d5f5b00-24d4-11eb-83b6-62c3c4ff7899.png)
